styles/styles.css

```
@@ -283,6 +356,8 @@ main {
 .button-oficial-site {
     text-decoration: none;
     color:#000000;
+    max-width: 115px;
+    margin-left: auto;
 }
```

```suggestion
button-official-site {
    text-decoration: none;
    color:#000000;
}
```
Ширина ссылки будет определяться картинкой. 
А margin не нужен, там будет gap и выравнивание гридами.